dotcom-rendering/src/layouts/AllEditorialNewslettersPageLayout.tsx

```diff
@@ -40,9 +40,7 @@ export const AllEditorialNewslettersPageLayout = ({
 
 	const renderAds = !isAdFreeUser;
 
-	const isInEuropeTest =
-		config.abTests.europeNetworkFrontVariant === 'variant' ||
-		config.switches['europeNetworkFrontSwitch'] === true;
+	const isInEuropeTest = config.switches['europeNetworkFrontSwitch'] === true;
```

Comment:
Minor non-blocking suggestion:
```suggestion
	const europeNetworkFrontSwitch = config.switches['europeNetworkFrontSwitch'] === true;
```

Or whatever seems more accurate - basically favouring naming of  'switch' over 'test'.

Reply:
Fair point but the switch will be removed right after this as well so I wanted to keep the diff as small as possible